components/molecules/NivoScatterChart/nivo-scatter-chart.tsx

```diff
@@ -142,7 +142,7 @@ const NivoScatterPlot = ({
           </>
         </ToggleGroup>
         {/* replaced display flex to hidden on show/bots container */}
-        <div className="flex flex-col gap-2 mt-3 md:mt-0 md:flex-row">
+        <div className="flex gap-2 mt-3 md:mt-0 flex-row">
```

Comment:
nit: We can omit `flex-row` since it's applied by default when using `flex`
```suggestion
        <div className="flex gap-2 mt-3 md:mt-0">
```

Reply:
Resolved in latest commit, Thanks :)